ICONPACK.md

```diff
@@ -1,5 +1,88 @@
-## Icon Pack list
-
-Here we describe all our SVG fragment identifiers. 
-
+ ## Icon Pack list
 
+  Here we describe all our SVG fragment identifiers. 
+  
+ :loudspeaker: **Disclaimer:** To use this fragments like an Icon in your apps, you should do like us in [IconCart](https://github.com/vtex-apps/store-icons/blob/master/react/IconCart.js).
+ 
+  
+<center>
+  
+| Icon                                    | ID                         |
+| --------------------------------------- | -------------------------- |
+| ![](svg/bnd-logo.svg)                   | bnd-logo                   |
+| ![](svg/hpa-arrow-from-bottom.svg)      | hpa-arrow-from-bottom      |
+| ![](svg/hpa-arrow-to-bottom.svg)        | hpa-arrow-to-bottom        |
+| ![](svg/hpa-arrow-back.svg)             | hpa-arrow-back             |
+| ![](svg/hpa-calendar.svg)               | hpa-calendar               |
+| ![](svg/hpa-cart.svg)                   | hpa-cart                   |
+| ![](svg/hpa-delete.svg)                 | hpa-delete                 |
+| ![](svg/hpa-play.svg)                   | hpa-play                   |
+| ![](svg/hpa-profile.svg)                | hpa-profile                |
+| ![](svg/hpa-save.svg)                   | hpa-save                   |
+| ![](svg/hpa-search.svg)                 | hpa-search                 |
+| ![](svg/hpa-hamburguer-menu.svg)        | hpa-hamburguer-menu        |
+| ![](svg/hpa-telemarketing.svg)          | hpa-telemarketing          |
+| ![](svg/mpa-filter-settings.svg)        | mpa-filter-settings        |
+| ![](svg/mpa-single-item.svg)            | mpa-single-item            |
+| ![](svg/mpa-gallery.svg)                | mpa-gallery                |
+| ![](svg/mpa-list-items.svg)             | mpa-list-items             |
+| ![](svg/mpa-heart.svg)                  | mpa-heart                  |
+| ![](svg/mpa-clone--outline.svg)         | mpa-clone--outline         |
+| ![](svg/mpa-clone--filled.svg)          | mpa-clone--filled          |
+| ![](svg/mpa-edit--outline.svg)          | mpa-edit--outline          |
+| ![](svg/mpa-edit--filled.svg)           | mpa-edit--filled           |
+| ![](svg/mpa-external-link--line.svg)    | mpa-external-link--line    |
+| ![](svg/mpa-external-link--outline.svg) | mpa-external-link--outline |
+| ![](svg/mpa-eyesight--outline--off.svg) | mpa-eyesight--outline--off |
+| ![](svg/mpa-eyesight--filled--off.svg)  | mpa-eyesight--filled--off  |
+| ![](svg/mpa-eyesight--outline--on.svg)  | mpa-eyesight--outline--on  |
+| ![](svg/mpa-eyesight--filled--on.svg)   | mpa-eyesight--filled--on   |
+| ![](svg/mpa-filter--outline.svg)        | mpa-filter--outline        |
```

Comment:
filter-outline and filled seems to be the same.

Reply:
Its because these two svg when are rendered its almost the same. Just copy the code and see [here](http://www.petercollingridge.appspot.com/svg-editor)

Reply:
Its important to remember that this svg fragment (almost all of them) was be made when we didn't had a defined architecture to render it, so it's possible that has an or other icon that not run perfectly.

Reply:
Can you fix this with @oldufre ?